src/renderer/components/+cluster-settings/status.tsx

```diff
@@ -27,6 +36,10 @@ export class Status extends React.Component<Props> {
             </TableRow>
           );
         })}
+        <TableRow key="kubeconfig">
```

Comment:
btw, `key`-prop usually required only inside items iteration.
read more: https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings

Reply:
right, removed that